insecure/attacknetwork/testHelpers.go

```diff
@@ -0,0 +1,90 @@
+package attacknetwork
```

Comment:
This file does not contain any new changes, just some shared test fixtures moved to this file for better organization.